highfidelity/hifi
# updates to overlay web browser

This PR adds a permissions bar and fixes QtWebEngine to work with HTML5 media apis (audio, video, etc) again.  Fixes pop up windows with the main browser  It also adds a version number to our UA for Chrome which makes many more sites functional and bumps the QtQuick version on the main browser to match other controls.

To test:

from "ctrl+b" browser only -- doesnt work with web entities

1. https://dev.windows.com/en-us/microsoft-edge/testdrive/demos/microphone/
2. click start demo
3. you should see the permissions bar.  choose 'allow.' 
4. You should see activity in the signal/frequency graphs
5. if u have a webcam:  https://webcamtoy.com/app/
6. go to www.asana.com.  click login with google.
7. a new window should pop up
